app/code/Magento/Bundle/Test/Unit/Model/LinkManagementTest.php

```diff
@@ -533,12 +542,15 @@ public function testAddChildCouldNotSave()
         $this->expectException(CouldNotSaveException::class);
 
         $productLink = $this->getMockBuilder(LinkInterface::class)
-            ->setMethods(['getSku', 'getOptionId', 'getSelectionId'])
+            ->setMethods(['getSku', 'getOptionId', 'getSelectionId', 'getWebSiteId'])
             ->disableOriginalConstructor()
             ->getMockForAbstractClass();
         $productLink->method('getSku')->willReturn('linked_product_sku');
         $productLink->method('getOptionId')->willReturn(1);
         $productLink->method('getSelectionId')->willReturn(1);
+        $store = $this->createMock(Store::class);
+        $this->storeManagerMock->expects($this->once())->method('getStore')->willReturn($store);
+        $store->expects($this->once())->method('getWebsiteId')->willReturn(100);
```

Comment:
This method doesn't exist anymore on the Link, please remove it. Let's adjust the test to represent the recent changes.

Reply:
The test will fail without this.

Reply:
Done